README.md

```diff
@@ -94,3 +94,93 @@ python -m unittest
 ```bash
 pylint app.py apitests models routes tests util
 ```
+## How to deploy
+This chapter describes how to get a production instance running.
+### Setup a new instance
+Clone the sources:
+```bash
+git clone https://github.com/buzzn/mybuzzn-backend.git
+```
+We recommend to setup a virtual env for the projects' requirements.
+```bash
+cd PROJECT_ROOT
+virtualenv -p `whereis python | cut -d ' ' -f3` .venv
+source .venv/bin/activate
+```
+Install the requirements
+```bash
+pip install -r "requirements.txt"
+pip install git+https://github.com/buzzn/discovergy.git
+```
+Create an endpoint in your webserver, for example for apache, create a new site:
+```
+<VirtualHost *:80>
+  ServerName your-domain.net
+  ErrorLog /path/to/error/log
+  CustomLog /path/to/access.log combined
+
+  WSGIDaemonProcess mybuzznbackend user=mybuzznbackend-user group=mybuzznbackend-group threads=5 python-path=/path/to/python:/and/path/to/packages
+  WSGIScriptAlias / /path/to/mybuzzn/projectdir/mybuzznbackend.wsgi
+  WSGIPassAuthorization On # Importent, so autho header is passed to the app
+
+    <Directory /path/to/project/>
+        WSGIProcessGroup mybuzznbackend
+        WSGIApplicationGroup %{GLOBAL}
+        Order deny,allow
+        Allow from all
+    </Directory>
+</VirtualHost>
+```
+Make sure that `WSGIPassAuthorization` is set to `On`, to have the app access to
+the auth headers.
+
+Make sure a file `setup_environment.py` exists in the project root, which sets
```

Comment:
"Make sure a file `setup_environment.py` exists in the project root which sets environment variables like this:"